src/CommunityToolkit.Maui/MauiCommunityToolkitOptions.cs

```diff
@@ -0,0 +1,28 @@
+using CommunityToolkit.Maui.Behaviors;
+using CommunityToolkit.Maui.Converters;
+
+namespace CommunityToolkit.Maui;
+
+/// <summary>
+/// .NET MAUI Community Toolkit Options.
+/// </summary>
+public class MauiCommunityToolkitOptions
```

Comment:
What do you think about moving this file to `CommunityToolkit.Maui.Core` and bringing the same functionality to `CommunityToolkit.Maui.Core.AppBuilderExtensions`? 

Then here in `CommunityToolkit.Maui`, we can add `CommunityToolkit.Maui.MauiCommunityToolkit.Options : CommunityToolkit.Maui.Core.MauiCommunityToolkitOptions`. 

I imagine that `CommunityToolkit.Maui.Core.AppBuilderExtensions` will be empty for now, so let's make it `public abstract class`, and then we have it implemented for whenever we need it in the future!

Reply:
In other words, let's keep the methods + properties you've already implemented here, but let's lay the framework for this same functionality in `CommunityToolkit.Maui.Core`

Reply:
I would prefer to keep it only in CommunityToolkit.Maui.
1. It is used only in this lib (and will be used only in this lib)
2. Properties that we have here (ThrowException) are specific to our implementation and may be unnecessary and not used in other implementations like Comet.

Or did you mean in the last sentence `CommunityToolkit.Maui.Core.MauiCommunityToolkitOptions`?

Reply:
Yup - let's keep everything here, but also add the same options functionality to `CommunityToolkit.Maui.Core`:

1. Create `CommunityToolkit.Maui.Core.MauiCommunityToolkit.Options`
  - It'll be an empty class for now until we add more properties
2. In `CommunityToolkit.Maui.Core.AppBuilderExtensions`, update `UseMauiCommunityToolkit`
   ```cs
   public static MauiAppBuilder UseMauiCommunityToolkit(this MauiAppBuilder builder, Action<MauiCommunityToolkitOptions>? options = default)
   ```
3. In `CommunityToolkit.Maui.MauiCommunityToolkitOptions`, inherit from `CommunityToolkit.Maui.MauiCommunityToolkitOptions`:
   ```cs
   public class MauiCommunityToolkitOptions : Core.MauiCommunityToolkitOptions
   ```
5. In `CommunityToolkit.Maui.AppBuilderExtensions`, pass `options`  into `UseMauiCommunityToolkitCore`:
   ```cs
   return builder.UseMauiCommunityToolkitCore(options);
   ```

Reply:
the problem #4 is that it is a different type `Argument type 'System.Action<CommunityToolkit.Maui.MauiCommunityToolkitOptions>?' is not assignable to parameter type 'System.Action<CommunityToolkit.Maui.Core.MauiCommunityToolkitOptions>?'`

Reply:
Good point! We'd need to cast it to `CommunityToolkit.Maui.Core.MauiCommunityToolkitOptions`.

Since the first inherits the other, it should be a valid cast, right? 

```cs
return builder.UseMauiCommunityToolkitCore((Core.MauiCommunityToolkitOptions)options);
```

Reply:
almost. we pass action. not sure if we can cast it to `Action<Core.Options>`

Reply:
but we pass action. and in this case, we need to pass argument Core.Options. while we already made some changes to the Maui.Options. let's pass Options instead of `Action<Options>` to Core

Reply:
But to be honest, as for me our order is not correct. we should call Core and then configure the lib